gtk4/Gir.toml

```diff
@@ -911,25 +910,35 @@ name = "Gtk.EventController"
 status = "generate"
 trust_return_value_nullability = false
 
-
 [[object]]
 name = "Gtk.EventControllerKey"
 status = "generate"
 trust_return_value_nullability = false
     [[object.signal]]
     name = "key-pressed"
     manual = true # to make use of gdk::keys::Key
+    inhibit = true
```

Comment:
This one is manual, so this one has no effect

Reply:
ok